examples/converting-from-json-to-user-defined-type/converting_from_json_to_user_defined_type.bal

```diff
@@ -8,10 +8,8 @@ type Coord record {
 public function main() {
     json j = {x: 1.0, y: 2.0};
 
-    // With mutable values, it would not be type-safe to allow a cast.
```

Comment:
Why was this removed?

Reply:
The same piece of text was repeated in both the description and comment sections. So I kept the description and changed the comment according to the slides

Reply:
When someone copies the example and runs it separately they won't see the description? Better to continue to have it IMO.